### ISSUE TITLE
Have you guys thought about how it can used to prevent Fake News on WhatsApp?

### ISSUE THREAD

**Issue by mateusbs17** [fake-news-detector/fake-news-detector#24]
I know that must be one of the most difficult obstacles to Fake News detection. But I would like to know if you guys have some ideas to bring these AI tools to help to map and fight Fake News spread via mobile interactions.

**Comment by rogeriochaves** [comment]
Yes, I believe whatsapp is the main source of misinformation nowadays so it should definitively be a priority to fight fake news there.

The two ideas we've thought so far are

1 - Having an app where you can use the "share" button from iOS and Android to export a selected message to the Fake News Detector App and it tells you if it seems fake or not

2 - Having a bot that you can forward the message, this is not possible for whatsapp right now (it looks like it will be with [WhatsApp Business](https://www.whatsapp.com/business/)), but we could already start something for Telegram for example and spread the bot to other messaging apps

Some people already told me they are interested in building both, but we haven't started anything yet.

Any extra ideas or will to help are very welcome ;D

**Comment by mateusbs17** [comment]
Nice, I didn't think about the first idea (and it looks like a nice MVP).

So far I thought about doing something like the Truecaller app. When you copy some text it scans and if it is a number it searches on its database and shows a popup to the user with the contact information of the number. But I don't know if it's a good approach, I think it's a little invasive.

It seems that mobile platforms have a lot of obstacles on this subject.

I am not a mobile developer but I am really interested in doing something to help this problem, and I am part of a team that have some mobile developers interested as well. Do you have any other discussion channel for this project?

**Comment by rogeriochaves** [comment]
@mateusbs17 sorry for the long delay, actually no, we don't have other channels but I believe I could create a telegram group and we could communicate there, what you think?

I'm personally interested in trying out React Native for building this app (at least the UI bits), and I have a friend that would also be insterested to help if we use RN, what you think?

**Comment by mateusbs17** [comment]
I like the idea of a Telegram group, let's do it. You can reach me at @mateusbs17.

I am inclined to learn more about React Native and Flutter, but Flutter is still on beta and the community is just getting started.

Let's do it.

**Comment by KingSujeet** [comment]
I have made the Whatsapp fake message detector app and we will upload it soon to google play store.

**Comment by rogeriochaves** [comment]
@KingSujeet cool! Let us know!
